lib/modules/components.js

```diff
@@ -364,17 +364,19 @@ export class _UrlSync extends React.Component {
       const currentQuery = extractQuery(self.location.search);
       const currentHash = {}; // TODO: address how hashes are displayed
       let pageIndex = -1;
+      let hist = {};
```

Comment:
random question. do we normally initialize variables this way? Can't we just declare it?